src/analysis/locally_convex/balanced_core_hull.lean

```diff
@@ -238,6 +238,13 @@ end
 
 variables (𝕜 E)
 
+lemma nhds_basis_balanced : (𝓝 (0 : E)).has_basis
+  (λ (s : set E), s ∈ 𝓝 (0 : E) ∧ balanced 𝕜 s) id :=
```

Comment:
Does `filter.has_basis_self` help to golf this proof?
Otherwise LGTM. Thanks! :tada:
bors d+